app/views/controllers/users/show/_profile.erb

```diff
@@ -63,7 +63,13 @@ footer =
       str, url, args = tab
       args ||= {}
       kwargs = merge_tab_args_with_extra_args(args, { class: "d-block" }) %>
-      <%= link_to(str, url, kwargs) %>
+      <% if args.keys.include?(:button) %>
+        <%= button_to(:destroy_object.t(type: :user), url, method: :delete,
+                      class: "d-block",
+                     ) %>
+      <% else %>
+        <%= link_to(str, url, kwargs) %>
+      <% end %>
```

Comment:
@nimmolo: `user_profile_tabs` includes information sufficient to produce a button.
But `link_to` can produce only links.
You might have a more elegant way of doing this.

Reply:
Oh, sorry. I missed this question. Just left the answer in a review.

Reply:
Perfect. That's exactly what I was looking for, but couldn't put my finger on it.